<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewSaleForm</class>
 <widget class="QWidget" name="NewSaleForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>423</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="horizontalSpacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="lblClient">
     <property name="text">
      <string>Client:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="4">
    <widget class="QLineEdit" name="txtClient">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QToolButton" name="btnSelectClient">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../res/resource.qrc">
       <normaloff>:/icons/select.png</normaloff>:/icons/select.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblProducts">
     <property name="text">
      <string>Products:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QToolButton" name="btnAddProduct">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../res/resource.qrc">
          <normaloff>:/icons/plus.png</normaloff>:/icons/plus.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnDeleteProduct">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../res/resource.qrc">
          <normaloff>:/icons/minus.png</normaloff>:/icons/minus.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="3" colspan="3">
    <widget class="QLineEdit" name="txtTotal">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="6">
    <widget class="QTableWidget" name="tableWidget">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <column>
      <property name="text">
       <string>Id</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Count</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Price</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Total price</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="4" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>630</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="2" colspan="2">
    <widget class="QPushButton" name="btnSale">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Sale</string>
     </property>
     <property name="icon">
      <iconset resource="../../res/resource.qrc">
       <normaloff>:/icons/buy.png</normaloff>:/icons/buy.png</iconset>
     </property>
    </widget>
   </item>
   <item row="4" column="4" colspan="2">
    <widget class="QPushButton" name="btnReset">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Reset</string>
     </property>
     <property name="icon">
      <iconset resource="../../res/resource.qrc">
       <normaloff>:/icons/delete.png</normaloff>:/icons/delete.png</iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="lblTotal">
     <property name="text">
      <string>Total:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../res/resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnSelectClient</sender>
   <signal>clicked()</signal>
   <receiver>NewSaleForm</receiver>
   <slot>slotButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>802</x>
     <y>12</y>
    </hint>
    <hint type="destinationlabel">
     <x>407</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnSale</sender>
   <signal>clicked()</signal>
   <receiver>NewSaleForm</receiver>
   <slot>slotButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>769</x>
     <y>443</y>
    </hint>
    <hint type="destinationlabel">
     <x>407</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnAddProduct</sender>
   <signal>clicked()</signal>
   <receiver>NewSaleForm</receiver>
   <slot>slotButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>13</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>407</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnDeleteProduct</sender>
   <signal>clicked()</signal>
   <receiver>NewSaleForm</receiver>
   <slot>slotButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>40</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>407</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnReset</sender>
   <signal>clicked()</signal>
   <receiver>NewSaleForm</receiver>
   <slot>slotButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>743</x>
     <y>408</y>
    </hint>
    <hint type="destinationlabel">
     <x>394</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotButton()</slot>
 </slots>
</ui>
